AIO/src/org/aio/gui/Gui.java

```diff
@@ -247,14 +245,37 @@ private TaskPanel addTask(final TaskType taskType) {
         if (taskPanel == null){
             throw new IllegalArgumentException(String.format("Task type %s not supported.", taskType.toString()));
         }
-
         JPopupMenu contextMenu = new JPopupMenu();
-        JMenuItem menuItem = new JMenuItem("Delete");
-        menuItem.addActionListener(e -> {
+        JMenuItem menuItemDelete = new JMenuItem("Delete");
+        JMenuItem menuItemSpace = new JMenuItem("");
```

Comment:
Use a `new JSeparator()` instead to get a small neat seperator.

Reply:
Never used the Java GUI before so yeah, thanks !